tests/runtest.sh

```diff
@@ -789,8 +794,27 @@ copy_test_native_bin_to_test_root
 load_unsupported_tests
 load_failing_tests
 
-if [ -n "$COMPlus_GCStress" ]; then
-    scriptPath=$(dirname $0)
+scriptPath=$(dirname $0)
+
+# Check if environment variables are provided
+if [ ! -z "$testEnv" ]; then
+    # Check if this is GC stress testing
+    hasGCStressEnv=`(source $testEnv; echo $COMPlus_GCStress)`
+    # Set __TestEnv that will be executed just before running tests
+    absTestEnvPath=`readlink -f ${testEnv}`
+    export __TestEnv='source '${absTestEnvPath}
+fi
+
+# Still support setting COMPlus_GCStress before runtest.sh but recommend
+# using --test-env option. 
+if [ -z "$hasGCStressEnv" ]; then
+    if [ -n "$COMPlus_GCStress" ]; then
+        hasGCStressEnv='true'
```

Comment:
Is there a reason we're using a string for 'true' (vs. false?) instead of booleans?